### PR TITLE
Implement stop all actions

### PR DESCRIPTION
**Pull request by jahazielhigareda**

### CODE REVIEW

File: src/Game/NeoServer.Game.Creatures/Model/Players/Player.cs [hunk 1]
```diff
@@ -917,6 +917,21 @@ public override ILoot DropLoot()
         return null;
     }
 
+    public void StopAllAction()
+    {
+        if (Attacking)
```
Comment (caioavidal) [inline src/Game/NeoServer.Game.Creatures/Model/Players/Player.cs]: I'd remove those ifs and call the 3 stops directly
StopAttack();
StopWalking();
StopFollowing();

File: src/Game/NeoServer.Game.Common/Contracts/Creatures/IPlayer.cs [hunk 2]
```diff
@@ -216,4 +216,5 @@ public interface IPlayer : ICombatActor, ISociableCreature
     void Read(IReadable readable);
     event ReadText OnReadText;
     void Write(IReadable readable, string text);
+    void StopAllAction();
```
Comment (caioavidal) [inline src/Game/NeoServer.Game.Common/Contracts/Creatures/IPlayer.cs]: Can you rename to "StopAllActions()" ?

File: tests/NeoServer.Game.Creatures.Tests/WalkableCreature/PlayerTest.cs [hunk 3]
```diff
@@ -296,4 +299,136 @@ public void PlayerDoesNotGainSkillsWhenUsingAHealingRune()
...
+
+        sut.OnStoppedWalking += _ => stoppedWalkEventEmitted = true;
+
+        var tile = new Mock<IDynamicTile>();
+        tile.Setup(x => x.Ground.StepSpeed).Returns(100);
+        tile.Setup(x => x.Location).Returns(new Location(100, 100, 7));
+
+        sut.SetCurrentTile(tile.Object);
+        //First instruction walking
+        sut.WalkTo(new Location(105, 100, 7));
```
Comment (caioavidal) [inline tests/NeoServer.Game.Creatures.Tests/WalkableCreature/PlayerTest.cs]: He we don't need WalkTo and instead Attack

Comment (jahazielhigareda) [reply tests/NeoServer.Game.Creatures.Tests/WalkableCreature/PlayerTest.cs]: This method is not use, i will remove

**Comment by caioavidal** [comment]
Looks good. thanks

**Comment by caioavidal** [comment]
close #233

**Pull request merged**